AuxTel/Troubleshooting/ATCS/AuxTel-Dome-Fails-to-Arrive-in-Position.rst

```diff
@@ -0,0 +1,187 @@
+    await self.process_as_completed(self.scheduled_coro)
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/remote_group.py", line 1173, in process_as_completed
+    raise e
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/remote_group.py", line 1170, in process_as_completed
+    ret_val = await res
+  File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-5.1.0/lib/python3.10/asyncio/tasks.py", line 571, in _wait_for_one
+    return f.result()  # May raise f.exception().
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1630, in wait_for_inposition
+    status = await asyncio.gather(*tasks)
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1696, in wait_for_atdome_inposition
+    await asyncio.wait_for(self.dome_az_in_position.wait(), timeout=timeout)
+  File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-5.1.0/lib/python3.10/asyncio/tasks.py", line 458, in wait_for
+    raise exceptions.TimeoutError() from exc
+  asyncio.exceptions.TimeoutError
+
+
+
+.. _AuxTel-Dome-Fails-to-Arrive-in-Position-Solution:
+
+Solution
+=========
+
+The dome tracking and positioning can be recovered from LOVE. 
+In case this fails, you can use a notebook to issue commands an recover the dome position as following. 
+All of the commands you will need can be found in the daytime_checkout notebook.
+
+
+A. **Procedure - Recovery from LOVE.**
+
+   1. State cycle ATDome through ``STANDBY`` and back to ``ENABLE`` from the ASummaryState. 
+      The transition must be quick enough so the dome shutter does not start closing and the recovery is faster; 
+      if it does close, the next step should deal with the shutter opening again.  
+      ``FAULT`` → ``STANDBY`` → ``START`` → ``DISABLED`` → ``ENABLED``.
+   
+      a. If ATPtg faulted, transition it back to enabled from ASummaryState. 
+
+   2. Select `auxtel/prepare_for/onsky` and move it to the beginning of the ATQueue. 
+      Execute this script to ensure the system is back and ready for observations, with the mirror cover and dome shutter opened, and the dome tracking enabled.
+
+
+   3. Keep ATQueue running to the next target. 
+      Confirmin LOVE that the dome is moving and following the mount.
+
+
+B. **Alternative procedure in case A. fails - Recovery using LOVE and a notebook.**
+
+   1. From the ASummaryState view on LOVE, transition the ATPtg back to enabled following the usual path: ``FAULT`` → ``STANDBY`` → ``START`` → ``DISABLED`` → ``ENABLED``
+   
+   2. Using an instantiated atcs class from a notebook (e.g. the daytime_checkout notebook), ensure that dome following is disabled and perform a dome slew (using a value for az that is at least 15 degrees away from where you are currently pointing) before reactivating `dome_following` by running the following commands:
+
```

Comment:
I am afraid that this part of the procedure needs to follow more closely the [original document](https://confluence.lsstcorp.org/display/OOD/ATDome+fails+to+arrive+in+position+-+ATPtg+in+FAULT+state). The text is already available as commented in lines 155-179. The python commands are not rendered in the [html document version](https://obs-ops.lsst.io/v/SITCOM-1448/AuxTel/Troubleshooting/ATCS/AuxTel-Dome-Fails-to-Arrive-in-Position.html).  Also @isotuela I am not sure about the access of the OS team to the notebooks. Maybe we can use this procedure to give more details about the notebook access.

Reply:
I think this line can be deprecated because, the new OS, we are no longer using the notebooks to command the telescope. They used to move the telescope with that script, but not anymore. This must be checked, or a solution must be found if the dome refuses to move.